discovery-provider/src/utils/default_underground_trending_tracks_strategy.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+from dateutil.parser import parse
+from src.utils.trending_strategy import TrendingType, TrendingVersion
+
+b = 5
+qw = 50
+hg = 1
+ie = 0.25
+pn = 0.01
+u = 30.0
+qq = 0.001
+oi = 20
+nb = 750
+
+class DefaultUndergroundTrendingTracksStrategy():
```

Comment:
I'm noticing that all of the `XTrendingTracksStrategy` classes follow the same basic pattern: set `trending_type` and `version` in init, and expose a `get_track_score` method. This tells me it should be encoded in our design in some way, rather than creating a new class definition for each trending strategy.

For problems like this we basically have two approaches - inheritance and composition. Inheritance has issues, but basically it would look like defining a base strategy class, and subclassing it various times to specialize the get_track_score method, etc. With a composition approach, you would accept get_track_score functions and type/version info via the constructor, and you'd just make different instances of the classes for different strategies.